### PR TITLE
Docs: link TypeScript docs from Calypso devdocs

### PR DESCRIPTION
**Pull request by razvanpapadopol**
#### Testing instructions
TypeScript docs should be accessible from the following pages: 
http://calypso.localhost:3000/devdocs/.github/CONTRIBUTING.md
http://calypso.localhost:3000/devdocs/docs/coding-guidelines.md

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Consistent coding style makes the code so much easier to read. Here are ours:
 
 * [All Coding Guidelines](../docs/coding-guidelines.md)
 	- [JavaScript](../docs/coding-guidelines/javascript.md)
+    - [TypeScript](../docs/coding-guidelines/typescript.md)
```
Comment (ockham) [inline .github/CONTRIBUTING.md]: I think we have a spaces vs tabs situation here :grimacing: 
```suggestion
	- [TypeScript](../docs/coding-guidelines/typescript.md)
```

Comment (sirreal) [inline .github/CONTRIBUTING.md]: Tabs! 😉 
```suggestion
	- [TypeScript](../docs/coding-guidelines/typescript.md)
```

**Review by ockham — APPROVED** [review]
One indentation related note, otherwise good to :ship: ! Thanks Razvan!

**Comment by sirreal** [comment]
Working well 👍 

I do notice that image URLs are broken in the GH docs 😞 , but that's not caused by this PR.

**Pull request merged**